docs/doc_check/CMakeLists.txt

```diff
@@ -1,13 +1,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
+set(check_cmd ${Python3_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/check.py)
+
 add_custom_target(check-doc
-  COMMAND ${Python3_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/check.py
+  COMMAND ${check_cmd}
     ${ONNX_MLIR_SRC_ROOT}
```

Comment:
At this point is there anything left in the src tree to check?

Reply:
check doc is used for various different script we list in our docs, so yes, it does verify things.

Reply:
IIRC, it checks .md files under the src root, in addition to the .dc files.

Reply:
> check doc is used for various different script we list in our docs, so yes, it does verify things.

@AlexandreEichenberger: I meant - are there files left in the source tree that are being verified after the two `.dc` files move to the bin root?

It sounds like the `.md` files are also verified separately?

Reply:
From check.py:

`for doc_file in chain(Path(root_dir).rglob('*.md'), Path(root_dir).rglob('*.dc'))`

Reply:
I think all of those are under `${ONNX_MLIR_SRC_ROOT}/docs`. Perhaps this could change to checking the `docs` dir and the binary dir?

Reply:
There are .md files in the root, though perhaps not ones we need to check. Is there a problem with continuing to glob from the src root?

Reply:
My thinking was if all the files that need checking live in only a couple of locations, they could be passed directly to simplify the expression and make it easier to track. It's not necessary, but it could be nice.